python/sglang/srt/reasoning_parser.py

```diff
@@ -0,0 +1,151 @@
+import re
+from typing import Dict
+
+REASONING_MODELS = ["deepseek-r1"]
```

Comment:
I think it's better to not duplicate the list of reasoning models. Removing `is_reasoning_model()` makes sense, I think it's cleaner to get the list of supported models from the `ReasoningParser` class though.
```suggestion
```